examples/UbidotsSaveValue/UbidotsSaveValue.ino

```diff
@@ -1,34 +1,42 @@
-#include <UbidotsArduinoGPRS.h>
-#include <SoftwareSerial.h> 
-
-//Serial Relay - Arduino will patch a 
-//serial link between the computer and the GPRS Shield
-//at 19200 bps 8-N-1
-//Computer is connected to Hardware UART
-//GPRS Shield is connected to the Software UART 
-#define APN "Put_the_APN_here" // Assign the APN 
-#define USER "Put_the_APN_user_herer"  // If your apn doesnt have username just put ""
-#define PASS "Put_the_APN_pwd_here"  // If your apn doesnt have password just put ""
-#define TOKEN "Put_your_Ubidots_token_here"  // Replace it with your Ubidots token
-#define VARIABLE_LABEL "temperature" // Assign the variable label 
-
-Ubidots client(TOKEN);
-SoftwareSerial gprs = SoftwareSerial(7, 8);
-SoftwareSerial *GPRSSerial = &gprs;
+// This example retrieves last value of a variable from the Ubidots API
+// using TCP protocol.
+
+/****************************************
+ * Include Libraries
+ ****************************************/
+#include <Ubidots.h>
+
+/****************************************
+ * Define Constants
+ ****************************************/
+// Your GPRS credentials, if any
+const char *APN  =           "Put_the_APN_here";
+const char *USER =           "Put_the_APN_user_here";
+const char *PASS =           "Put_the_APN_pwd_here";
+const char *TOKEN=           "Put_your_Ubidots_token_here"; 
+const char *DEVICE_LABEL=    "Put_your_device_API_label"; 
+const char *VARIABLE_LABEL=  "Put_your_variable_API_label";
+
+Ubidots client(TOKEN,APN,USER,PASS);
+
+
+/****************************************
+ * Auxiliar Functions
+ ****************************************/
+
+// Put here your auxiliar functions
+
+/****************************************
+ * Main Functions
+ ****************************************/
 
 void setup() {
   Serial.begin(115200);
-  GPRSSerial->begin(19200);
-  if (! client.init(*GPRSSerial)) {
-    Serial.println(F("Couldn't find FONA"));
-    while (1);
-  }
-  client.setApn(APN,USER,PASS);
-  //client.setDebug(false);
+  client.setDebug(true); // Set true to make available debug messages
 }
 
 void loop() {
   float value = analogRead(A0);  // Reading analog pin A0
   client.add(VARIABLE_LABEL, value);  
-  client.sendAll();
-}
\ No newline at end of file
+  client.send();
```

Comment:
capture here the send() return to know if the values were sent.

Apply this to ALL the examples